File: 2358-number-of-ways-to-split-array/number-of-ways-to-split-array.py
class Solution:
    def waysToSplitArray(self, nums: List[int]) -> int:
        s = sum(nums)
        result = 0
        l = 0
        for i in range(0, len(nums) - 1):
            l += nums[i] 
            result += 1 if l >= s - l else 0
        return result

File: 2358-number-of-ways-to-split-array/number-of-ways-to-split-array.cpp
class Solution {
public:
    int waysToSplitArray(vector<int>& nums) {
        long s = 0;
        for (int num: nums)
            s += num;
        long l = 0;
        int result = 0;
        for (int i = 0; i < nums.size() - 1; i++){
            l += nums[i];
            result += (l >= (s - l)) ? 1 : 0;
        }
        return result;
    }
};